src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -85,7 +85,8 @@ const ProfilePicture = ({
                   testId="images"
                   style={styles.profileImage(size, width)}
                   source={{
-                    uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH
+                    uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH,
+                    cache: 'web'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, changing the `cache` property to `'web'` could lead to performance issues. Consider using `'force-cache'` instead.
```diff
- cache: 'web'
+ cache: 'force-cache'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->